dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/internal/ConfigurationType.java

```diff
@@ -18,6 +18,7 @@
 
 public enum ConfigurationType {
     activeBuildProfiles,
+    aggregatorParentBaseDir,
```

Comment:
Are both 'aggregatorParentBaseDir' and 'aggregatorParentBasedir' needed?

Reply:
If they are I think it would be good to add a comment documenting why. I initially thought it was a mistake.